### PR TITLE
Update test status for innodb-java-reader

### PR DESCRIPTION
**Pull request by shunfan-shao**
When I tried to claim the projects to fix, I found that these tests in the project have been fixed. I have tested them under NonDex with master branch.

`com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testGenFillingRateHeatmap`
`com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testGenLsnHeatmap`
`com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testQueryAllDirectIO`

### CODE REVIEW

**Comment by shunfan-shao** [comment]
Logs when running with: 
```sh
$ git log $(find -name InnodbReaderBootstrapTest.java)
commit 4e829225439c02ac7553bcde3dcee9220ef3088d
Author: xu <xu.zx@alibaba-inc.com>
Date:   Thu Jun 4 11:14:34 2020 +0800

    Add testcase for quotation in cli module.

commit 7c1014979ab10aaf263a2eab721ef00e5ce9c415
Author: xu <xu.zx@alibaba-inc.com>
Date:   Fri May 22 18:12:37 2020 +0800

    Support specify secondary key root page number. If secondary key root page is wrong, but key type matches, when no pk found, throw an exception. Add charset and collate toString method in TableDef. Support specify CHARSET (already support CHARACTER SET) in column.

commit 7ddfe75b7b39c225040cc2a805ef4c99ea9cf9fc
Author: xu <xu.zx@alibaba-inc.com>
Date:   Wed May 20 14:43:06 2020 +0800

    Support secondary querying with projection and ordering. Update test for year and date.

commit 4df363f98592839c4628e61193a9da5c5cc5901a
Author: xu <xu.zx@alibaba-inc.com>
Date:   Thu May 14 20:50:41 2020 +0800

    Support desc ordering. Refine corner case for comparison comparator. Fix typo of setFullyQualifiedName. Empty table test.

commit 707ad7395fb298b0066983048a426ffdc94cf6e0
Author: xu <xu.zx@alibaba-inc.com>
Date:   Mon May 4 23:00:05 2020 +0800

    Support projection.

commit c3c52e8b7109e27767c75080685975ad1d529936
Author: xu <xu.zx@alibaba-inc.com>
Date:   Fri May 1 15:39:02 2020 +0800

    Enhance range query to support ComparisonOperator.

commit 30357fb9807f1ce07678a6c7b79ec175efbc8a64
Author: xu <xu.zx@alibaba-inc.com>
Date:   Wed Apr 29 23:52:54 2020 +0800

    Support composite primary key and no primary key. Rename package to com.alibaba.database. Refactor key comparison. Support record iteration lazily. Refine core method - readRecord to support compsite key on leaf page and non-leaf page. Make TableReader an interface. Add numerous test cases to verify composite key.

commit 5b94740e5fe538167ff0f7f41c170d3725fe0b3d
Author: xu <xu.zx@alibaba-inc.com>
Date:   Thu Apr 23 17:23:50 2020 +0800

    Upgrade alibaba p3c version to 3.0.6, update checkstyle.xml, make max line length to 120 and format.

commit 0ca07bd7803fa5589ce466426314101e4353b68b
Author: xu <xu.zx@alibaba-inc.com>
Date:   Wed Jan 29 14:59:45 2020 +0800

    1. Add testcase for innodb-heatmap and innodb-java-reader-cli module.
    2. Support mmap, buffer, direct io for cli when dumping file.
    3. Support dump with header.
    4. Support dump by specifying delimiter.
```

File: pr-data.csv [hunk 1]
```diff
@@ -93,9 +93,9 @@ https://github.com/alibaba/fastjson,e05e9c5e4be580691cc55a59f3256595393203a1,.,c
...
-https://github.com/alibaba/innodb-java-reader,24a95d8005d480d460c3004fad89cfa295b9cbb4,innodb-java-reader,com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testQueryAllDirectIO,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/23
+https://github.com/alibaba/innodb-java-reader,24a95d8005d480d460c3004fad89cfa295b9cbb4,innodb-java-reader-cli,com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testGenFillingRateHeatmap,ID,DeveloperFixed,,https://github.com/TestingResearchIllinois/idoft/issues/23
+https://github.com/alibaba/innodb-java-reader,24a95d8005d480d460c3004fad89cfa295b9cbb4,innodb-java-reader-cli,com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testGenLsnHeatmap,ID,DeveloperFixed,,https://github.com/TestingResearchIllinois/idoft/issues/23
+https://github.com/alibaba/innodb-java-reader,24a95d8005d480d460c3004fad89cfa295b9cbb4,innodb-java-reader-cli,com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest.testQueryAllDirectIO,ID,DeveloperFixed,,https://github.com/TestingResearchIllinois/idoft/issues/23
```
Comment (darko-marinov) [inline pr-data.csv:98]: The module name should be changed, but (1) the tests are OD not ID, and (2) checks all tests in com.alibaba.innodb.java.reader.InnodbReaderBootstrapTest (at least the seven already in pr-data.csv).
